contrib/images/ibcsim-bcd/setup-bcd.sh

```diff
@@ -243,7 +243,7 @@ NETWORK="regtest"
 BTC_CONFIRMATION_DEPTH=1
 CHECKPOINT_FINALIZATION_TIMEOUT=2
 BABYLON_TAG="01020304"
-CONSUMER_NAME="test-consumer"
+CONSUMER_NAME="07-tendermint-0"
```

Comment:
I see the reason we need to change it to cosumer id is because it is needed in [reward distribution](https://github.com/babylonlabs-io/cosmos-bsn-contracts/blob/136933c503b3dbadf0b6b01fa71692204d855e55/contracts/babylon/src/contract.rs#L341). If the case, it seems that we need to rename the config item to consumer id and make it non-optional. cc @SebastianElvis

Reply:
Here `CONSUMER_NAME` is consumer name, not consumer ID, so it does not need to be `"07-tendermint-0"`.

It's actually not clear to me why Babylon relies on Cosmos BSN to provide the consumer ID in the payload. Shouldn't Babylon get this information from `x/btcstkconsumer` or IBC channel module @RafilxTenfen ?

Reply:
Modified at https://github.com/babylonlabs-io/babylon/pull/1489